docs/getting-started.md

```diff
@@ -153,7 +154,13 @@ We'll want to provide a sample JSON file that the Shunter application can proxy
 }
 ```
 
-Notice the `layout.template` property in the JSON we just created? That indicates the [Dust](http://www.dustjs.com/) template that should be used to render the page. If we open up [http://localhost:5400/home](http://localhost:5400/home) now, then you'll see an error in your terminal indicating that the "home" template can't be found.
+Notice the `layout.template` property in the JSON we just created? That indicates the [Dust](http://www.dustjs.com/) template that should be used to render the page.
+
+The `layout.namespace` property is required by Dust and the value can be any string.
```

Comment:
My memory's a bit hazy on this, but I _believe_ `namespace` is a shunter-specific thing, not Dust-specific.

Reply:
I had a quick look and there's certainly stuff in shunter for it;
https://github.com/springernature/shunter/blob/8d33fcc19bb5ac58aa99362823e2eb2cffb00f01/lib/dust.js#L10

https://github.com/springernature/shunter/blob/ddc20a596851c094d15853f662861e42e8e9efdd/lib/renderer.js#L259

Whether it could be removed from shunter IDK lol, @moddular ?

Reply:
I have a suspicion this isn't used anymore - I think it was used when we had themes - as a series of locations used when looking up templates.